### PR TITLE
Reduce schema reuse and promote dependencies to top level directory [Part 5]

### PR DESCRIPTION
**Pull request by jvc9109**
## Summary
The goal is to reduce excessive schema reuse in the scope of cleaning up the api-definitions files.
Cleaned subschemas (not referenced from the paths ) are moved to top level directory.

## Links
[Part 4](https://github.com/Rebilly/api-definitions/pull/1508)

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Organization.yaml [hunk 1]
```diff
@@ -83,9 +83,119 @@ properties:
...
+    description: Organization questionnaire.
+    properties:
+      role:
+        type: string
+        nullable: true
+        description: Role of the owner.
+      monthlyTransactions:
+        type: string
+        nullable: true
+        description: Amount of monthly processed transaction.
```
Comment (noon-dawg) [inline openapi/components/schemas/Organization.yaml:97]: nitpick: I think this should be plural
```suggestion
        description: Amount of monthly processed transactions.
```

File: openapi/components/schemas/Rules/Actions/offer-purchase-bump.yaml [hunk 2]
```diff
@@ -1,7 +1,45 @@
...
+            - name
+            - weight
+            - offers
+            - choices
+          properties:
+            name:
+              type: string
+              description: |-
+                Name of the bump offer version.
+                This field is useful to measure split tests.
```
Comment (noon-dawg) [inline openapi/components/schemas/Rules/Actions/offer-purchase-bump.yaml:23]: suggestion:
```suggestion
                This field is useful when measuring split tests.
```

**Review by noon-dawg — DISMISSED** [review]
All good on the docs side

**Comment by jvc9109** [comment]
@Rebilly/js @Rebilly/php @Rebilly/docs  please review

File: openapi/components/schemas/UpcomingInvoice.yaml [hunk 3]
```diff
@@ -34,7 +34,7 @@ properties:
     type: integer
   organizationId:
     allOf:
-      - $ref: ./Organizations/OrganizationId.yaml
+      - $ref: ./OrganizationId.yaml
   items:
     type: array
     description: Upcoming Invoice items array.
```
Comment (noon-dawg) [inline openapi/components/schemas/UpcomingInvoice.yaml:40]: nitpick: capitalization 
```suggestion
    description: Upcoming invoice items array.
```

**Review by noon-dawg — APPROVED** [review]
Looks good on the docs side

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**